app/views/api/plans/_default_plan.html.slim

```diff
@@ -1,3 +1,4 @@
+/ DEPRECATED: remove in favor of javascript_pack_tag 'default_plan_selector'. app/views/api/application_plans/_listing.html.slim
```

Comment:
WDYM with deprecated? The file should be removed?

Reply:
Yes, all plan selectors will be replaced by this one and this file removed (in another PR or PRs).